actions/addContract.ts

```diff
@@ -41,6 +42,12 @@ const _addContract: ActionFn = async (context: Context, event: Event) => {
   // Process the logs
   let hasErrors = false;
   for (const log of transactionEvent.logs) {
+    // Do not process logs that are not from a `ComposableCoW`-compatible contract
+    // This is a *normal* case, if the contract is not `ComposableCoW`-compatible
+    // then we do not need to do anything, and therefore don't flag as an error.
+    if (!isCompatible(await provider.getCode(log.address))) {
```

Comment:
If Composable CoW is known, why not checking the address. 
Do we care about other instances of the contracts?

Reply:
The check would be faster, and wouldn't give you false positives, right?

Reply:
Yes, to both. Ultimately it comes down to the question of if we want to run a generic watch tower, or not. Given that others may end up piggy-backing on our infrastructure (and this leads to more cows), I think it's a fair thing to support. An example here is Swapr and their Stackly product, that was using our old watch Web3 Actions (which we weren't aware of).